src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -16,6 +17,7 @@
  */
 public abstract class AddCommand extends UndoableCommand {
 
+    public static final AutoRescheduleManager AUTO_RESCHEDULE_MANAGER = AutoRescheduleManager.getInstance();
```

Comment:
Is there any reason why the ```AUTO_RESCHEDULE_MANAGER``` is in this class? I feel that since this is only used in the ```AddEventCommand```, it should be stored in the ```AddEventCommand```.

Reply:
That's because I'm planning to converge AddEventCommand, AddTaskCommand, AddReminderCommand to a generic AddCommand.